lib/ansible/modules/windows/win_updates.py

```diff
@@ -58,6 +58,24 @@
         - This is only used if C(reboot=yes) and a reboot is required.
         default: 1200
         version_added: '2.5'
+    serverselection:
```

Comment:
Instead of replicating the serverselection options for end users we should be trying to give an easier way of defining it. I see choices 0 to 2 are quite useful for people today but 3 is probably not any use right now. I think we should do the following for this option;

* Change the option to be `server_selection` to fit the standard we have for longer module options
* Change the type to `str`
* Set the choices to `default`, `managed_server`, `windows_update` (or something similar)
* Have the default value be `default`

In the win_updates.ps1 code, change the string to whatever int value it represents. The reason for this is having a string and a limited choice set makes it a lot easier for people not familiar with WUA to use the module.